website/src/components/Changelog/GuiChangeItem.tsx

```diff
@@ -0,0 +1,28 @@
+import Entry from "./Entry";
+import Link from "next/link";
+
+export default function GuiChangeItem({
+  enable,
+  pull,
+  children,
+}: {
+  enable: boolean;
+  pull: string;
+  children: React.ReactNode;
```

Comment:
You could use Optional types and default `enable = true` by default. Then you could generalize this component to just `ChangeItem` and we can use it across all things (linking to a PR is a nice feature)